### DISCUSSION TITLE
Trace input is null when passing in nodes, or Documents with transformations to VectorStoreIndex constructor

### DISCUSSION THREAD

**Discussion by mwertheim-zco** [langfuse/langfuse#1538] [Support]
I am creating an instance of llama_index VectoreStoreIndex using the class method `from_documents`. 
If I pass in a list of Documents, the trace appears normal with input showing document metadata.
If I pass in a list of TextNodes, or a list of Documents with a list of transformations (to produce text nodes inside the constructor) the input shows null.
Is this normal and expected behavior?
langfuse v2.21.1
python v3.10.13
llama-index v0.10.20

This is the trace using a list of Documents and no transformations. It appears detailed.
<img width="1297" alt="Screenshot 2024-03-28 at 6 22 28 PM" src="https://github.com/langfuse/langfuse/assets/147201130/81a6ae80-660f-4fee-9e07-f0b86b28aa75">

This is the trace using either a list of Nodes OR a list of Documents AND a list of transformations.
<img width="1318" alt="Screenshot 2024-03-28 at 6 23 06 PM" src="https://github.com/langfuse/langfuse/assets/147201130/e2d56038-84e8-4914-93b3-702dac71adf3">

Any insight is appreciated

**Comment by marcklingen** [comment]
Thanks for sharing and creating a bug issue (#1551), will look into it
